### PR TITLE
adds and rebalances items

### PR DESCRIPTION
**Pull request by Dorodomki**
# About the pull request

adds a vest, medkit, hardhat for the mt's and the CE closets to help them out in the shift, limiting their gear as FOB reinforcements since any mt was able to grab a marine armor and fight as one, here i give them some gear to not make them look like a marine and have some differences in case they get deployed on FOB

adds one pouch slot more to the civilian m3 vest to leave it at the same storage level of other marine armors and making it a bit more useful

# Explain why it's good for the game

adds some basic gear for the mt's being a medkit to fill the medkit pouch they have, a civilian vest if they are required in the FOB and a welder chestrig since they had to hack a surplus vendor to get them plus a blue hardhat to cover their heads and because it combines with the uniform colour

# Changelog

:cl:
add: adds a welder chestrig, basic medkit, civilian vest, blue hardhat to the mt's and chief engineer closets
balance: rebalances the civilian vest having one pouch
/:cl:

### CODE REVIEW

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Those vests aren't for deployment purposes so no go on that.

Secondly, it has one storage slot because it's pretty damn fast, I see no reason to buff it.

**Comment by Drulikar** [comment]
In the changelog template, it intentionally has each line preceded with a type e.g. `add: blah blah` or `balance: blah blah` those are needed to generate the changelog because they indicate what kind of change it is. I have edited your changelog to reflect this, but should you change the scope of the pr be sure to update it.

**Review by morrowwolf — CHANGES_REQUESTED** [review]
No vests. They are USCM personnel, give them armor if you want to give them armor. Keep the hazard vest in there as well.

**Comment by Dorodomki** [comment]
> No vests. They are USCM personnel, give them armor if you want to give them armor. Keep the hazard vest in there as well.

they aren't marines to have the same gear as them, im thinking of more "civilian or old gear" used by them since they are engineers after all, their armor should be oriented towards ballistic protection instead of slashes and full body protection, that's my point in giving them ballistic vests

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm [hunk 1]
```diff
@@ -27,6 +29,8 @@
 	new /obj/item/storage/pouch/general/medium(src)
 	new /obj/item/storage/pouch/construction(src)
 	new /obj/item/storage/pouch/tools(src)
+	new /obj/item/storage/pouch/medkit(src)
```
Comment (morrowwolf) [inline code/game/objects/structures/crates_lockers/closets/secure/engineering.dm:32]: No medkit pouch

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Otherwise fine

File: code/game/objects/structures/crates_lockers/closets/secure/engineering.dm (hunk 1 above)
Comment (Dorodomki) [reply code/game/objects/structures/crates_lockers/closets/secure/engineering.dm:32]: done now merg

**Review by morrowwolf — CHANGES_REQUESTED** [review]
Nothing new int he chief engineer locker.

Put all of that stuff in the chief engineer vendor.

All of the new stuff placed in the engineer locker needs to be in the non-groundside section so we don't find random marine gear and such in civilian engineer lockers. Really there should be two different ones but I won't make you do that.

**Comment by Dorodomki** [comment]
> Nothing new in the chief engineer locker.
> 
> Put all of that stuff in the chief engineer vendor.
> 
> All of the new stuff placed in the engineer locker needs to be in the non-groundside section so we don't find random marine gear and such in civilian engineer lockers. Really there should be two different ones but I won't make you do that.

morrow they literally get an universal welder chestrig, medkit and nothing else what marine gear are you talking about dude, they aren't getting any marine gear.

**Comment by morrowwolf** [comment]
Feel free to reopen when changes are complete.

**Comment by Dorodomki** [comment]
this pr just added medkits, overalls, blue hazard helmets and a welder chestrig to the MT, CE lockers, deleted what you wanted and you really know how to motive people to developt for the game, im gonna do the changes for the last time because is something really cool develop here

**Pull request closed without merging**